crates/chainio/clients/avsregistry/src/writer.rs

```diff
@@ -330,19 +333,190 @@ impl AvsRegistryChainWriter {
         info!(tx_hash = ?tx,"succesfully deregistered operator with the AVS's registry coordinator" );
         Ok(*tx.tx_hash())
     }
+
+    /// Sets minimum stake for the quorum
+    ///
+    /// Can only be caled by registry coordinator's owner.
+    /// The quorum numbers must exist, else the tx will fail.
+    /// # Arguments
+    ///
+    /// # `quorum_numbers` - The respective quorum numbers in [`u8`].
+    /// # `minimum_stake` - The minimum stake in [`U96`].
+    ///
+    /// # Returns
+    ///
+    /// * [`TxHash`] - The transaction hash of the transaction.
+    pub async fn set_minimum_stake_for_quorum(
+        &self,
+        quorum_numbers: u8,
+        minimum_stake: U96,
+    ) -> Result<TxHash, AvsRegistryError> {
+        let provider = get_signer(&self.signer.clone(), &self.provider);
+
+        let contract_stake_registry = StakeRegistry::new(self.stake_registry_addr, provider);
+        let reg_coordinator_owner =
+            RegistryCoordinator::new(self.registry_coordinator_addr, get_provider(&self.provider))
+                .owner()
+                .call()
+                .await?
+                ._0;
+        let caller_address =
+            get_signer(&self.signer.clone(), &self.provider).default_signer_address();
+        if !reg_coordinator_owner.eq(&caller_address) {
```

Comment:
Great idea! We should change this in a future PR to be a single call and match the returned error.

Reply:
👍